Userland/Libraries/LibJS/Runtime/RegExpPrototype.cpp

```diff
@@ -103,6 +103,88 @@ static void increment_last_index(GlobalObject& global_object, Object& regexp_obj
     regexp_object.set(vm.names.lastIndex, Value(last_index), true);
 }
 
+// 1.1.2.1 Match Records, https://tc39.es/proposal-regexp-match-indices/#sec-match-records
+struct MatchRecord {
+    MatchRecord(regex::Match const& match)
```

Comment:
Unless it collides with something, let's remove the record suffix for consistency.

See:

- 44221756ab7515617450400bf23d0e8df486c2b7
- https://github.com/SerenityOS/serenity/blob/9bc3ad75b44cc63bcbdb85d3f7c1c21b8735f9e7/Userland/Libraries/LibJS/Runtime/PromiseReaction.h#L15-L26